libs/designer-ui/src/lib/connectorsummarycard/index.tsx

```diff
@@ -47,19 +42,19 @@ export const ConnectorSummaryCard = (props: ConnectorSummaryCardProps) => {
           description={description}
           style={isCard ? undefined : { marginRight: '8px' }}
           innerAriaHidden="true"
+          className={mergeClasses('msla-recommendation-panel-card-visible-on-hover', 'info-dot-visible-on-hover')}
+        />
+        <FavoriteButton
+          connectorId={connector.id}
+          operationId={undefined}
```

Comment:
NIT: do we need to send it if its undefined?

Reply:
We don't necessary need it, but its to diffrentiate whether or not this is a Connector Favorite button or an Operation Favorite Button